corehq/motech/openmrs/repeater_helpers.py

```diff
@@ -226,7 +227,20 @@ def get_subresource_instances(requests, person_uuid, subresource):
     )).json()['results']
 
 
-def get_patient(requests, info, openmrs_config, problem_log):
+def find_patient(domain, requests, case_id, openmrs_config):
+    PatientFinder = to_function(openmrs_config.patient_finder)
+    if not PatientFinder:
+        return None
```

Comment:
Can this return a message/exception that the `finder` was not found? Combining this with `None` when no patients are found would not make it clear why there were no records found.